// @vialabs-io/npm-registry/src/chainConfigs.ts
// Auto-generated chain configurations
// DO NOT EDIT DIRECTLY - Edit files in src/chains/{mainnet,testnet} instead

import { ChainConfig, ChainConfigs } from './types';

export const mainnetConfigs: ChainConfigs = {
  "1": {
    "name": "Ethereum Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1,
    "message": "0x7b67dF6728E294db2eb173ac7c738a4627Ae5e11",
    "feeToken": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
    "weth": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
    "usdc": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
    "eurc": "0x1aBaEA1f7C830bD89Acc67eC4af516284b1bC33c",
    "rpc": "https://1.rpc.vialabs.io/",
    "explorer": "https://etherscan.io/",
    "circleTokenMessenger": "0xbd3fa81b58ba92a82136038b25adec7066af3155",
    "circleMessageTransmitter": "0x0a992d191deec32afe36203ad87d7d289a738f81",
    "circleTokenMinter": "0xc4922d64a24675e16e1586e3e3aa56c06fabe907",
    "isCCTPEnabled": true,
    "featureGateway": "0xba30eFc4F2553d0A7733D2CA3cEc98cDC4D428ba",
    "featureCCTP": "0x31f840D4cFB05F2B72C41B501d6dd1CFe4106aa6",
    "protoCCTPGateway": "0x53f67b67418dcFB5ca88D443ee82584148b3c973",
    "intentCCTPGateway": "0x0000000000000000000000000000000000000000",
    "cctpLeafChains": [
      {
        "chainId": 48900,
        "rootManager": "0x3012C9175ef181Fb8Da827cc439cd88861cf6aaB",
        "leafManager": "0x3fBD26bd2F4ea70042634ac10972C3E3DeaCd875"
      }
    ]
  },
  "10": {
    "name": "Optimism Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 10,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85",
    "weth": "0x4200000000000000000000000000000000000006",
    "usdc": "0x0b2c639c533813f4aa9d7837caf62653d097ff85",
    "circleTokenMessenger": "0x2B4069517957735bE00ceE0fadAE88a26365528f",
    "circleMessageTransmitter": "0x4d41f22c5a0e5c74090899e5a8fb597a8842b3e8",
    "isCCTPEnabled": true,
    "featureGateway": "0xfDdE8946A0A1755b3a74afE7b3383dEe4fb5712f",
    "featureCCTP": "0xE3dEb61797983F859f1E1D3671F0EB7Ad97333a9",
    "protoCCTPGateway": "0x64541cE2aa06194D59c4D130435792c1f178f750",
    "intentCCTPGateway": "0x0000000000000000000000000000000000000000"
  },
  "14": {
    "name": "Flare Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 14,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0x1D80c49BbBCd1C0911346656B529DF9E5c2F783d",
    "rpc": "https://14.rpc.vialabs.io/",
    "explorer": "https://flare-explorer.flare.network/",
    "featureGateway": "0x2224aC62D65453A2317b995fC084e966e4BAC2d5"
  },
  "19": {
    "name": "Songbird Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 19,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0xb36e1df3A03d860c6f70731847dFD64E04B0dDB7",
    "featureGateway": "0xc6ff38E3dA2B10318E728ec5c097Bed97C1801D8"
  },
  "25": {
    "name": "Cronos Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 25,
    "message": "0x2224ac62d65453a2317b995fc084e966e4bac2d5",
    "feeToken": "0xc21223249CA28397B4B6541dfFaEcC539BfF0c59",
    "weth": "0x5C7F8A570d578ED84E63fdFA7b1eE72dEae1AE23",
    "rpc": "https://25.rpc.vialabs.io/",
    "explorer": "https://cronos.org/explorer/"
  },
  "30": {
    "name": "Rootstock Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 30,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0x542fDA317318eBF1d3DEAf76E0b632741A7e677d"
  },
  "37": {
    "name": "XPLA EVM Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 37,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x14f5f510B42D371352c605c0c1B972409b82c0F6",
    "weth": "0x26A72EE66E2A9dBEC659Ba86d9683e26CEDBdE9c",
    "featureGateway": "0x96C8b0297d1a07a31Ea402Bc750aA2BD59c873f8"
  },
  "40": {
    "name": "Telos EVM Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 40,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x8712796136Ac8e0EEeC123251ef93702f265aa80",
    "weth": "0xD102cE6A4dB07D247fcc28F366A623Df0938CA9E",
    "usdc": "0x8712796136Ac8e0EEeC123251ef93702f265aa80",
    "featureGateway": "0xA10953dEFac127d1b7A645f81045aB5127893124",
    "protoCCTPGateway": "0xd6d9317E95Ca1cEDe3D06F3D6844a739194212cc"
  },
  "42": {
    "name": "Lukso Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 42,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0xF9506457473bFcc285a22C7Ea42e88ABA2b888EE",
    "featureGateway": "0x39869675cCC8B39Eb8016C2Cb7a718B119442b56"
  },
  "50": {
    "name": "XDC Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 50,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x49d3f7543335cf38Fa10889CCFF10207e22110B5",
    "weth": "0x951857744785E80e2De051c32EE7b25f9c458C42",
    "rpc": "https://50.rpc.vialabs.io/",
    "explorer": "https://explorer.xinfin.network/"
  },
  "56": {
    "name": "Binance Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 56,
    "message": "0x7b67dF6728E294db2eb173ac7c738a4627Ae5e11",
    "feeToken": "0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d",
    "weth": "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c",
    "rpc": "https://56.rpc.vialabs.io/",
    "explorer": "https://bscscan.com/"
  },
  "66": {
    "name": "OKXChain Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 66,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xc946DAf81b08146B1C7A8Da2A851Ddf2B3EAaf85",
    "weth": "0x8F8526dbfd6E38E3D8307702cA8469Bae6C56C15",
    "featureGateway": "0x9E0f7B09576D1F0cBfF1f834A8e5bcc3Db1EF804"
  },
  "82": {
    "name": "Meter Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 82,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xDF63b7B8D0330e20a69f557Bac54943b76D0a7cf",
    "weth": "0x160361ce13ec33C993b5cCA8f62B6864943eb083"
  },
  "88": {
    "name": "Viction Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 88,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x0000000000000000000000000000000000000000",
    "weth": "0x381B31409e4D220919B2cFF012ED94d70135A59e"
  },
  "100": {
    "name": "Gnosis Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 100,
    "message": "0x75c0223A8f9f8f3baB3FbC6d151381C3C576Ba36",
    "feeToken": "0xDDAfbb505ad214D7b80b1f830fcCc89B60fb7A83",
    "weth": "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d",
    "rpc": "https://100.rpc.vialabs.io/",
    "explorer": "https://gnosisscan.io/"
  },
  "108": {
    "name": "Thundercore Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 108,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x22e89898A04eaf43379BeB70bf4E38b1faf8A31e",
    "weth": "0x413cEFeA29F2d07B8F2acFA69d92466B9535f717",
    "featureGateway": "0xCeD17226078Bcb0B54006Eff802192AFcF97356a"
  },
  "122": {
    "name": "Fuse Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 122,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x620fd5fa44BE6af63715Ef4E65DDFA0387aD13F5",
    "weth": "0x0BE9e53fd7EDaC9F859882AfdDa116645287C629",
    "featureGateway": "0xfDdE8946A0A1755b3a74afE7b3383dEe4fb5712f"
  },
  "137": {
    "name": "Polygon Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 137,
    "message": "0x1C5800eb5fECB7760D7F1978ad744feA652a7b27",
    "feeToken": "0x3c499c542cEF5E3811e1192ce70d8cC03d5c3359",
    "weth": "0x0d500B1d8E8eF31E21C99d1Db9A6444d3ADf1270",
    "usdc": "0x3c499c542cef5e3811e1192ce70d8cc03d5c3359",
    "rpc": "https://137.rpc.vialabs.io/",
    "explorer": "https://polygonscan.com/",
    "circleTokenMessenger": "0x9daF8c91AEFAE50b9c0E69629D3F6Ca40cA3B3FE",
    "circleMessageTransmitter": "0xF3be9355363857F3e001be68856A2f96b4C39Ba9",
    "circleTokenMinter": "0x10f7835F827D6Cf035115E10c50A853d7FB2D2EC",
    "isCCTPEnabled": true,
    "featureGateway": "0x9bf255FB6C7bbB0e060Ed6CC8996edFB0c4f5e0b",
    "featureCCTP": "0x34d786a3cA75c11cb941840992De0Df2Eb706ad7",
    "protoCCTPGateway": "0x996fCc660B3dF10d547A3A79A75191f1E344c2cb",
    "intentCCTPGateway": "0x0000000000000000000000000000000000000000"
  },
  "146": {
    "name": "Sonic Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 146,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x0000000000000000000000000000000000000000",
    "weth": "0x039e2fB66102314Ce7b64Ce5Ce3E5183bc94aD38"
  },
  "148": {
    "name": "Shimmer EVM Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 148,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xeCE555d37C37D55a6341b80cF35ef3BC57401d1A",
    "weth": "0xBEb654A116aeEf764988DF0C6B4bf67CC869D01b",
    "featureGateway": "0x39869675cCC8B39Eb8016C2Cb7a718B119442b56"
  },
  "169": {
    "name": "Manta Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 169,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xb73603C5d87fA094B7314C74ACE2e64D165016fb",
    "weth": "0x0Dc808adcE2099A9F62AA87D9670745AbA741746",
    "featureGateway": "0xA10953dEFac127d1b7A645f81045aB5127893124"
  },
  "185": {
    "name": "Mint Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 185,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xb62F35B9546A908d11c5803ecBBA735AbC3E3eaE",
    "weth": "0x4200000000000000000000000000000000000006",
    "featureGateway": "0xc6ff38E3dA2B10318E728ec5c097Bed97C1801D8"
  },
  "196": {
    "name": "X-Layer Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 196,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xA8CE8aee21bC2A48a5EF670afCc9274C7bbbC035",
    "weth": "0xe538905cf8410324e03a5a23c1c177a474d59b2b",
    "rpc": "https://196.rpc.vialabs.io/",
    "explorer": "https://www.okx.com/web3/explorer/xlayer"
  },
  "199": {
    "name": "BitTorrent Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 199,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xCa424b845497f7204D9301bd13Ff87C0E2e86FCF",
    "weth": "0x23181F21DEa5936e24163FFABa4Ea3B316B57f3C"
  },
  "204": {
    "name": "opBNB Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 204,
    "message": "0x1274E183AA921972531414Df700E673dE1Be878D",
    "feeToken": "0x9e5AAC1Ba1a2e6aEd6b32689DFcF62A509Ca96f3",
    "weth": "0x4200000000000000000000000000000000000006",
    "rpc": "https://240.rpc.vialabs.io/",
    "explorer": "https://opbnbscan.com/"
  },
  "207": {
    "name": "Vinuchain Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 207,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xc0264277fcca5fcfabd41a8bc01c1fcaf8383e41",
    "weth": "0xed8c5530a0a086a12f57275728128a60dff04230",
    "featureGateway": "0x96C8b0297d1a07a31Ea402Bc750aA2BD59c873f8"
  },
  "213": {
    "name": "B2network Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 213,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x681202351a488040Fa4FdCc24188AfB582c9DD62",
    "weth": "0x4200000000000000000000000000000000000006"
  },
  "250": {
    "name": "Fantom Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 250,
    "message": "0x72E052Fa7f0788e668965d37B6c38C88703B7859",
    "feeToken": "0x28a92dde19D9989F39A49905d7C9C2FAc7799bDf",
    "weth": "0x21be370D5312f44cB42ce377BC9b8a0cEF1A4C83",
    "rpc": "https://250.rpc.vialabs.io/",
    "explorer": "https://ftmscan.com/"
  },
  "252": {
    "name": "Fraxtal Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 252,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xDcc0F2D8F90FDe85b10aC1c8Ab57dc0AE946A543",
    "weth": "0xFC00000000000000000000000000000000000006",
    "featureGateway": "0xC705addd35aBa54f17c9E2cf772F829f2f4D9876"
  },
  "255": {
    "name": "Kroma Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 255,
    "message": "0xf8B8656Ce65Ecf334AcCe299e24E97fB5069c2C8",
    "feeToken": "0x0257e4d92C00C9EfcCa1d641b224d7d09cfa4522",
    "weth": "0x4200000000000000000000000000000000000001",
    "featureGateway": "0x96C8b0297d1a07a31Ea402Bc750aA2BD59c873f8"
  },
  "288": {
    "name": "Boba Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 288,
    "message": "",
    "feeToken": "0x66a2A913e447d6b4BF33EFbec43aAeF87890FBbc",
    "weth": "0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000",
    "rpc": "https://288.rpc.vialabs.io/",
    "explorer": "https://bobascan.com/"
  },
  "321": {
    "name": "KCC Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 321,
    "message": "0xf8B8656Ce65Ecf334AcCe299e24E97fB5069c2C8",
    "feeToken": "0x0039f574eE5cC39bdD162E9A88e3EB1f111bAF48",
    "weth": "0x4446Fc4eb47f2f6586f9fAAb68B3498F86C07521",
    "featureGateway": "0x96C8b0297d1a07a31Ea402Bc750aA2BD59c873f8"
  },
  "369": {
    "name": "Pulse Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 369,
    "message": "0xba01F2bA548e69bA26Fd06a3bdf1A7857eeAC435",
    "feeToken": "0x15d38573d2feeb82e7ad5187ab8c1d52810b1f07",
    "weth": "0xa1077a294dde1b09bb078844df40758a5d0f9a27",
    "rpc": "https://369.rpc.vialabs.io/",
    "explorer": "https://scan.mypinata.cloud/ipfs/bafybeih3olry3is4e4lzm7rus5l3h6zrphcal5a7ayfkhzm5oivjro2cp4/#/"
  },
  "388": {
    "name": "Cronos zkEVM Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 388,
    "message": "0x205B6E9a868F26DC18feE372071C5FA42a6Cea3D",
    "feeToken": "0xaa5b845F8C9c047779bEDf64829601d8B264076c",
    "weth": "0xc1bf55ee54e16229d9b369a5502bfe5fc9f20b6d"
  },
  "416": {
    "name": "SX Network Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 416,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xe2aa35C2039Bd0Ff196A6Ef99523CC0D3972ae3e",
    "weth": "0xaa99bE3356a11eE92c3f099BD7a038399633566f",
    "featureGateway": "0xCeD17226078Bcb0B54006Eff802192AFcF97356a"
  },
  "530": {
    "name": "FXCore Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 530,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": ""
  },
  "570": {
    "name": "Rollux Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 570,
    "message": "0xf0861412E46E34DaF7A1fe3fe805e7f713cA3bD9",
    "feeToken": "0x368433cac2a0b8d76e64681a9835502a1f2a8a30",
    "weth": "0x4200000000000000000000000000000000000006",
    "rpc": "https://570.rpc.vialabs.io/",
    "explorer": "https://explorer.rollux.com/"
  },
  "592": {
    "name": "Astar Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 592,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x6a2d262d56735dba19dd70682b39f6be9a931d98",
    "weth": "0xaeaaf0e2c81af264101b9129c00f4440ccf0f720",
    "featureGateway": "0x13482A6cFA571Aa9F8c68768f84d465b82598eb0"
  },
  "820": {
    "name": "Callisto Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 820,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xbf6c50889d3a620eb42C0F188b65aDe90De958c4",
    "weth": "0xF5AD6F6EDeC824C7fD54A66d241a227F6503aD3a",
    "featureGateway": "0xC705addd35aBa54f17c9E2cf772F829f2f4D9876"
  },
  "841": {
    "name": "Taraxa Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 841,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0x5d0Fa4C5668E5809c83c95A7CeF3a9dd7C68d4fE"
  },
  "888": {
    "name": "Wanchain Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 888,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x52A9CEA01c4CBDd669883e41758B8eB8e8E2B34b",
    "weth": "0xdabD997aE5E4799BE47d6E69D9431615CBa28f48"
  },
  "911": {
    "name": "Taproot Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 911,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": ""
  },
  "995": {
    "name": "5ire Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 995,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0xCFaEB74409E4C6756C43F75455fc42A6A3FdEb1f",
    "usdc": "0x85DAc390d61cdaCb346701cC4948870A704d3f4a",
    "protoCCTPGateway": "0xC5c36314540cFaA48Fd71BA1BB9BB3966d04a8AC"
  },
  "1030": {
    "name": "Conflux eSpace Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1030,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x8f0fb159380176d324542b3a7933f0c2fd0e2d5f",
    "weth": "0x14b2d3bc65e74dae1030eafd8ac30c533c976a9b",
    "usdc": "0xfe97e85d13abd9c1c33384e796f10b73905637ce",
    "rpc": "https://1030.rpc.vialabs.io/",
    "explorer": "https://evm.confluxscan.net/"
  },
  "1088": {
    "name": "Metis Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1088,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xEA32A96608495e54156Ae48931A7c20f0dcc1a21",
    "weth": "0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000",
    "rpc": "https://1088.rpc.vialabs.io/",
    "explorer": "https://explorer.metis.io/"
  },
  "1101": {
    "name": "PolygonZK Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1101,
    "message": "0x39869675cCC8B39Eb8016C2Cb7a718B119442b56",
    "feeToken": "0xA8CE8aee21bC2A48a5EF670afCc9274C7bbbC035",
    "weth": "0x4F9A0e7FD2Bf6067db6994CF12E4495Df938E6e9",
    "rpc": "https://1101.rpc.vialabs.io/",
    "explorer": "https://www.polygonzkscan.com/"
  },
  "1111": {
    "name": "Wemix Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1111,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xE3F5a90F9cb311505cd691a46596599aA1A0AD7D",
    "weth": "0x7D72b22a74A216Af4a002a1095C8C707d6eC1C5f"
  },
  "1116": {
    "name": "Core Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1116,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xa4151b2b3e269645181dccf2d426ce75fcbdeca9",
    "weth": "0x40375c92d9faf44d2f9db9bd9ba41a3317a2404f",
    "featureGateway": "0x2224aC62D65453A2317b995fC084e966e4BAC2d5"
  },
  "1135": {
    "name": "Lisk Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1135,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0x4200000000000000000000000000000000000006"
  },
  "1231": {
    "name": "Ultron Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1231,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x3c4e0fded74876295ca36f62da289f69e3929cc4",
    "weth": "0x3a4f06431457de873b588846d139ec0d86275d54",
    "featureGateway": "0x96C8b0297d1a07a31Ea402Bc750aA2BD59c873f8"
  },
  "1234": {
    "name": "Step Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1234,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xE3F5a90F9cb311505cd691a46596599aA1A0AD7D",
    "weth": "0xb58A9d5920AF6aC1a9522B0B10F55dF16686D1b6"
  },
  "1284": {
    "name": "Moonbeam Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1284,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xFFFFFFFF7D2B0B761Af01Ca8e25242976ac0aD7D",
    "weth": "0xAcc15dC74880C9944775448304B263D191c6077F",
    "featureGateway": "0x440AA59AC28460a43537a2eA892d96Acd8eaBdb0"
  },
  "1285": {
    "name": "Moonriver Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1285,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xE3F5a90F9cb311505cd691a46596599aA1A0AD7D",
    "weth": "0x98878B06940aE243284CA214f92Bb71a2b032B8A",
    "featureGateway": "0x39869675cCC8B39Eb8016C2Cb7a718B119442b56"
  },
  "1329": {
    "name": "Sei Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1329,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x3894085Ef7Ff0f0aeDf52E2A2704928d1Ec074F1",
    "weth": "0xE30feDd158A2e3b13e9badaeABaFc5516e95e8C7",
    "featureGateway": "0xe3b3274bb685F37C7f17a604039c77a6A16Cfc2a"
  },
  "1339": {
    "name": "Elysium Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1339,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x0000000000000000000000000000000000000000",
    "weth": "0xd80ef77b0289732e13d1769850b5a70ecc196777"
  },
  "1456": {
    "name": "Zkbase Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1456,
    "message": "0x205B6E9a868F26DC18feE372071C5FA42a6Cea3D",
    "feeToken": "0x0935D1e004eFd2300CB2511cA118A67C274221B3",
    "weth": "0x678a8587B2bB8566aF7b4f4Fe154686e4DD4E487"
  },
  "1490": {
    "name": "Vitruveo Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1490,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xbCfB3FCa16b12C7756CD6C24f1cC0AC0E38569CF",
    "weth": "0xC0C0A38067Ba977676AB4aFD9834dB030901bE2d",
    "rpc": "https://1490.rpc.vialabs.io/",
    "explorer": "https://explorer.vitruveo.xyz/",
    "protoCCTPGateway": "0xd6d9317E95Ca1cEDe3D06F3D6844a739194212cc"
  },
  "1559": {
    "name": "Tenet Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1559,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x55c32e8aB56021C012AB3d5a586b743664E73DA0",
    "weth": "0xd6cb8a253e12893b0cF39Ca78F7d858652cCa1fe",
    "featureGateway": "0xCeD17226078Bcb0B54006Eff802192AFcF97356a"
  },
  "1625": {
    "name": "Gravity Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1625,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0xBB859E225ac8Fb6BE1C7e38D87b767e95Fef0EbD",
    "featureGateway": "0xCeD17226078Bcb0B54006Eff802192AFcF97356a"
  },
  "1689": {
    "name": "Nero Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1689,
    "protoCCTPGateway": "0xdD7f4dE4F625D51AE39541d1AC67967878A4A98c",
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x8712796136Ac8e0EEeC123251ef93702f265aa80",
    "usdc": "0x8712796136Ac8e0EEeC123251ef93702f265aa80",
    "weth": "0xd540c9cfa1aecb92c649d14d46715b7974b3ac7f"
  },
  "1750": {
    "name": "Metal Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1750,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0x4200000000000000000000000000000000000006",
    "featureGateway": "0xC705addd35aBa54f17c9E2cf772F829f2f4D9876"
  },
  "1777": {
    "name": "Gauss Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1777,
    "message": "0xfDdE8946A0A1755b3a74afE7b3383dEe4fb5712f",
    "feeToken": "0x2baFd05252ADcA1143b77a88A72E1022B59ad379",
    "weth": "0xb45fC65405B1a28Bb24AF49fe2caa278525Fe977",
    "rpc": "https://1777.rpc.vialabs.io/",
    "explorer": "https://www.explorer.gaussgang.com/"
  },
  "1890": {
    "name": "Lightlink Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1890,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0x81A1f39f7394c4849E4261Aa02AaC73865d13774"
  },
  "1996": {
    "name": "Sanko Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1996,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x13D675BC5e659b11CFA331594cF35A20815dCF02",
    "weth": "0xE01e3b20C5819cf919F7f1a2b4C18bBfd222F376",
    "featureGateway": "0x39869675cCC8B39Eb8016C2Cb7a718B119442b56"
  },
  "1997": {
    "name": "Kyoto Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1997,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "",
    "rpc": "https://1997.rpc.vialabs.io/",
    "explorer": "https://kyotoscan.io"
  },
  "2000": {
    "name": "Dogechain Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 2000,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x765277EebeCA2e31912C9946eAe1021199B39C61",
    "weth": "0xB7ddC6414bf4F5515b52D8BdD69973Ae205ff101"
  },
  "2016": {
    "name": "Mainnetz Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 2016,
    "message": "0x1274E183AA921972531414Df700E673dE1Be878D",
    "feeToken": "0x0000000000000000000000000000000000000000",
    "weth": "0x41c515fA7D541bCbDEfB44f0FE2B1629aec140b9",
    "rpc": "https://59144.rpc.vialabs.io/",
    "explorer": "https://explorer.mainnetz.io/"
  },
  "2222": {
    "name": "Kava Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 2222,
    "message": "0x9E0f7B09576D1F0cBfF1f834A8e5bcc3Db1EF804",
    "feeToken": "0x919C1c267BC06a7039e03fcc2eF738525769109c",
    "weth": "0xc86c7C0eFbd6A49B35E8714C5f59D99De09A225b",
    "rpc": "https://2222.rpc.vialabs.io/",
    "explorer": "https://atomscan.com/kava/"
  },
  "2649": {
    "name": "Ailayer Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 2649,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": ""
  },
  "2911": {
    "name": "Hychain Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 2911,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": ""
  },
  "3068": {
    "name": "Bifrost Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 3068,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x640952E7984f2ECedeAd8Fd97aA618Ab1210A21C",
    "weth": "0x1c1b06405058AbE02e4748753aeD1458BEFEE3B9",
    "featureGateway": "0xc6ff38E3dA2B10318E728ec5c097Bed97C1801D8"
  },
  "3776": {
    "name": "Astar zkEVM Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 3776,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xa8ce8aee21bc2a48a5ef670afcc9274c7bbbc035",
    "weth": "0xe9cc37904875b459fa5d0fe37680d36f1ed55e38",
    "featureGateway": "0x2224aC62D65453A2317b995fC084e966e4BAC2d5"
  },
  "3797": {
    "name": "AlveyChain Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 3797,
    "message": "0x65EEc58ef38882422E887B82f7085e9a9C35dCA1",
    "feeToken": "0x7a0b5E86d2C1e873E005780419eeE16ba741808a",
    "weth": "0xCb3e9919C56efF1004E54175a01e39163a352129",
    "rpc": "https://3797.rpc.vialabs.io/",
    "explorer": "https://alveyscan.com/"
  },
  "4061": {
    "name": "Nahmii Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 4061,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0x4200000000000000000000000000000000000006"
  },
  "4078": {
    "name": "Muster Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 4078,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0x869Bf8814d77106323745758135b999D34C79a87"
  },
  "4337": {
    "name": "Beam Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 4337,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x76BF5E7d2Bcb06b1444C0a2742780051D8D0E304",
    "weth": "0xD51BFa777609213A653a2CD067c9A0132a2D316A",
    "featureGateway": "0x08615FA517d47a9FF8C19F53D5Cbfd2Dd14b554E"
  },
  "4689": {
    "name": "IoTeX Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 4689,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x6de8aCC0D406837030CE4dd28e7c08C5a96a30d2",
    "weth": "0x4fffe0168b04f039496ca5189f9596c22693A19c",
    "rpc": "https://4689.rpc.vialabs.io/",
    "explorer": "https://iotexscan.io/"
  },
  "5000": {
    "name": "Mantle Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 5000,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x09Bc4E0D864854c6aFB6eB9A9cdF58aC190D0dF9",
    "weth": "",
    "rpc": "https://5000.rpc.vialabs.io/",
    "explorer": "https://mantlescan.xyz/"
  },
  "5101": {
    "name": "Syndicate Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 5101,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": ""
  },
  "6699": {
    "name": "OX Chain Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 6699,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0xe0009848520C305C92eB13EAdd5676095f82Ab5f",
    "featureGateway": "0xc6ff38E3dA2B10318E728ec5c097Bed97C1801D8"
  },
  "6900": {
    "name": "Nibiru Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 6900,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x08EBA8ff53c6ee5d37A90eD4b5239f2F85e7B291",
    "usdc": "0x08EBA8ff53c6ee5d37A90eD4b5239f2F85e7B291",
    "protoCCTPGateway": "0x844f9248EA80Ee65F633a7Fa82Af78643d63834C",
    "weth": "0x1429B38e58b97de646ACd65fdb8a4502c2131484"
  },
  "7000": {
    "name": "Zeta Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 7000,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x05BA149A7bd6dC1F937fA9046A9e05C05f3b18b0",
    "weth": "0x5F0b1a82749cb4E2278EC87F8BF6B618dC71a8bf",
    "featureGateway": "0x6EeFCF3fdd03471dbb0dd984C90EA8f678c266Df"
  },
  "7171": {
    "name": "BitRock Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 7171,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xD647a6E1F898c39bA594443c19De0872EA935310",
    "weth": "0x413f0E3A440abA7A15137F4278121450416882d5",
    "rpc": "https://7171.rpc.vialabs.io/",
    "explorer": "https://explorer.bit-rock.io/"
  },
  "7332": {
    "name": "Horizon Eon Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 7332,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xCc44eB064CD32AAfEEb2ebb2a47bE0B882383b53",
    "weth": "0xF5cB8652a84329A2016A386206761f455bCEDab6",
    "featureGateway": "0x39869675cCC8B39Eb8016C2Cb7a718B119442b56"
  },
  "7560": {
    "name": "Cyber Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 7560,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x81759adbf5520ad94da10991dfa29ff147d3337b",
    "weth": "0x14778860e937f509e651192a90589de711fb88a9"
  },
  "7700": {
    "name": "Canto Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 7700,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x80b5a32E4F032B2a058b4F29EC95EEfEEB87aDcd",
    "weth": "0x826551890Dc65655a0Aceca109aB11AbDbD7a07B",
    "rpc": "https://7700.rpc.vialabs.io/",
    "explorer": "https://cantoscan.com/"
  },
  "8217": {
    "name": "Klaytn Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 8217,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xe2053bcf56d2030d2470fb454574237cf9ee3d4b",
    "weth": "0xe4f05a66ec68b54a58b17c22107b02e0232cc817",
    "featureGateway": "0x2224aC62D65453A2317b995fC084e966e4BAC2d5"
  },
  "8453": {
    "name": "Base Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 8453,
    "message": "0xe3b3274bb685F37C7f17a604039c77a6A16Cfc2a",
    "feeToken": "0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913",
    "weth": "0x4200000000000000000000000000000000000006",
    "usdc": "0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913",
    "rpc": "https://8453.rpc.vialabs.io/",
    "explorer": "https://basescan.org/",
    "circleTokenMessenger": "0x1682Ae6375C4E4A97e4B583BC394c861A46D8962",
    "circleMessageTransmitter": "0xAD09780d193884d503182aD4588450C416D6F9D4",
    "circleTokenMinter": "0xe45B133ddc64bE80252b0e9c75A8E74EF280eEd6",
    "isCCTPEnabled": true,
    "featureGateway": "0x9A9b54a7b63361743531C52E5f7fa2d4BD3fD33e",
    "featureCCTP": "0x29451F0ee3Ef946b49598232928E98b30c6f255A",
    "protoCCTPGateway": "0x804FD8228bc5A02db6CdA3fFa96a9C6b6D49b1e7",
    "intentCCTPGateway": "0x0000000000000000000000000000000000000000",
    "cctpLeafChains": [
      {
        "chainId": 40,
        "rootManager": "0xbEc2085d80E71969625E8b497Bc3d66a2f665D99",
        "leafManager": "0x9e0517ADAEc365Af770F67F7e83055d60Ad7D4dc"
      }
    ]
  },
  "8866": {
    "name": "Lumio Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 8866,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x1C93569537a52c144b6B24640F72d74b6c1B0f3C",
    "weth": "0x4200000000000000000000000000000000000006",
    "featureGateway": "0x96C8b0297d1a07a31Ea402Bc750aA2BD59c873f8"
  },
  "9980": {
    "name": "Combo Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 9980,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x7c6b91d9be155a6db01f749217d76ff02a7227f2",
    "weth": "0x4200000000000000000000000000000000000006"
  },
  "10001": {
    "name": "EthereumPOW Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 10001,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x2AD7868CA212135C6119FD7AD1Ce51CFc5702892",
    "weth": "0x7Bf88d2c0e32dE92CdaF2D43CcDc23e8Edfd5990"
  },
  "11501": {
    "name": "Bevm Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 11501,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xa67ED736649F2958A35fd249a584151056b4b745",
    "weth": "0xB5136FEba197f5fF4B765E5b50c74db717796dcD"
  },
  "12324": {
    "name": "L3X Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 12324,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0xD3f8b9D33b159E8f5141d28880b216d31B00ee63",
    "featureGateway": "0x08615FA517d47a9FF8C19F53D5Cbfd2Dd14b554E"
  },
  "12553": {
    "name": "RSS3 Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 12553,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0xE27D019909738D98Ab7F850C05eE07806c30c71D",
    "featureGateway": "0xc6ff38E3dA2B10318E728ec5c097Bed97C1801D8"
  },
  "13371": {
    "name": "Immutable zkEVM Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 13371,
    "message": "",
    "feeToken": "0x6de8aCC0D406837030CE4dd28e7c08C5a96a30d2",
    "weth": "0x52A6c53869Ce09a731CD772f245b97A4401d3348",
    "rpc": "https://13371.rpc.vialabs.io/"
  },
  "13939": {
    "name": "reef-mainnet",
    "network": "mainnet",
    "type": "reef",
    "chainId": 13939,
    "message": "0xdAFcF1e328c09cbc480A09f751c170CfCca3e083",
    "messageSubstrate": "5EMjsd1648TVgor7qD19pXda8mbfbDZ3NyTRUiWKZBRmPZ7e",
    "protoCCTPGateway": "0x54A83CE0165988C4259Ce5ecF6F17692a0700A9F",
    "feeToken": "0xCA2917E6625E158be8066e8D8a584B022AB6D8D6",
    "usdc": "0xCA2917E6625E158be8066e8D8a584B022AB6D8D6",
    "weth": "0x0000000000000000000000000000000001000000"
  },
  "18686": {
    "name": "MXC Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 18686,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0xcBCE60BAD702026d6385E5f449e44099A655d14f",
    "featureGateway": "0x82FEa67e31f43D6f6C8858604433Ee469fbB1CF2"
  },
  "22776": {
    "name": "Map Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 22776,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0x13cb04d4a5dfb6398fc5ab005a6c84337256ee23",
    "featureGateway": "0xc6ff38E3dA2B10318E728ec5c097Bed97C1801D8"
  },
  "23294": {
    "name": "Oasis Sapphire Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 23294,
    "message": "0x5E9DBAC1d92b04E5Bd8Bd42845B315568b969Da0",
    "feeToken": "0xE48151964556381B33f93E05E36381Fd53Ec053E",
    "weth": "0x8Bc2B030b299964eEfb5e1e0b36991352E56D2D3",
    "rpc": "https://23294.rpc.vialabs.io/",
    "explorer": "https://www.oasisscan.com/"
  },
  "32520": {
    "name": "Bitgert Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 32520,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x0000000000000000000000000000000000000000",
    "weth": "0x0eb9036cbE0f052386f36170c6b07eF0a0E3f710",
    "featureGateway": "0x82FEa67e31f43D6f6C8858604433Ee469fbB1CF2"
  },
  "32659": {
    "name": "Fusion Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 32659,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x8a20c13b42d7fe418f10f922f2cee06246c24269",
    "weth": "0x8fdc02dc969a22c02ddffd3d8b547fab3d7702fe",
    "featureGateway": "0xc6ff38E3dA2B10318E728ec5c097Bed97C1801D8"
  },
  "33979": {
    "name": "Funkichain Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 33979,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0x4200000000000000000000000000000000000006",
    "featureGateway": "0x96C8b0297d1a07a31Ea402Bc750aA2BD59c873f8"
  },
  "34443": {
    "name": "Mode Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 34443,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xd988097fb8612cc24eeC14542bC03424c656005f",
    "weth": "0x4200000000000000000000000000000000000006",
    "featureGateway": "0x75c0223A8f9f8f3baB3FbC6d151381C3C576Ba36"
  },
  "42161": {
    "name": "Arbitrum Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 42161,
    "message": "0x65EEc58ef38882422E887B82f7085e9a9C35dCA1",
    "feeToken": "0xaf88d065e77c8cC2239327C5EDb3A432268e5831",
    "weth": "0x82aF49447D8a07e3bd95BD0d56f35241523fBab1",
    "usdc": "0xaf88d065e77c8cC2239327C5EDb3A432268e5831",
    "rpc": "https://42161.rpc.vialabs.io/",
    "explorer": "https://arbiscan.io/",
    "circleTokenMessenger": "0x19330d10D9Cc8751218eaf51E8885D058642E08A",
    "circleMessageTransmitter": "0xC30362313FBBA5cf9163F0bb16a0e01f01A896ca",
    "circleTokenMinter": "0xE7Ed1fa7f45D05C508232aa32649D89b73b8bA48",
    "isCCTPEnabled": true,
    "featureGateway": "0x88eC0180180Af475430eb6e8F38981cb0b61F93d",
    "featureCCTP": "0x5a596d81cF8911818F3231eb87a77Ac1426485Bf",
    "protoCCTPGateway": "0x3b05FC65F04489538619EBCe0661f29597DA8df2",
    "intentCCTPGateway": "0x0000000000000000000000000000000000000000",
    "cctpLeafChains": [
      {
        "chainId": 660279,
        "rootManager": "0x482fEC83988a57f7B853EBE3927Edbd49DBF44BC",
        "leafManager": "0x9e0517ADAEc365Af770F67F7e83055d60Ad7D4dc"
      },
      {
        "chainId": 995,
        "rootManager": "0x56f6f2A23AE5EF82D8Dc46934F7Cf8D0b15ddEdD",
        "leafManager": "0xA5895cB2F28579a1B002c524DB7A5250EE6A5773"
      },
      {
        "chainId": 6900,
        "rootManager": "0x30af901F0F031Da450AB6f0AB695185690E10c91",
        "leafManager": "0x1c0e7F1967aB417e7363112500d770BCCBf18BbD"
      },
      {
        "chainId": 1689,
        "rootManager": "0x4C1e3c449Eb89EbB4D3813C6ccf618Ea8Ee48E25",
        "leafManager": "0x9e0517ADAEc365Af770F67F7e83055d60Ad7D4dc"
      }
    ]
  },
  "42170": {
    "name": "Arbitrum Nova Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 42170,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x750ba8b76187092B0D1E87E28daaf484d1b5273b",
    "weth": "0x722E8BdD2ce80A4422E880164f2079488e115365",
    "featureGateway": "0x6EeFCF3fdd03471dbb0dd984C90EA8f678c266Df"
  },
  "42220": {
    "name": "Celo Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 42220,
    "message": "0xe3b3274bb685F37C7f17a604039c77a6A16Cfc2a",
    "feeToken": "0xcebA9300f2b948710d2653dD7B07f33A8B32118C",
    "weth": "0x471EcE3750Da237f93B8E339c536989b8978a438",
    "usdc": "0xcebA9300f2b948710d2653dD7B07f33A8B32118C",
    "rpc": "https://42220.rpc.vialabs.io/",
    "explorer": "https://explorer.celo.org/mainnet/"
  },
  "42262": {
    "name": "Oasis Emerald Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 42262,
    "message": "0xe3b3274bb685F37C7f17a604039c77a6A16Cfc2a",
    "feeToken": "0xdC19A122e268128B5eE20366299fc7b5b199C8e3",
    "weth": "0x21C718C22D52d0F3a789b752D4c2fD5908a8A733",
    "rpc": "https://42262.rpc.vialabs.io/",
    "explorer": "https://www.oasisscan.com/"
  },
  "43114": {
    "name": "Avalanche Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 43114,
    "message": "0x72E052Fa7f0788e668965d37B6c38C88703B7859",
    "feeToken": "0xB97EF9Ef8734C71904D8002F8b6Bc66Dd9c48a6E",
    "weth": "0xB31f66AA3C1e785363F0875A1B74E27b85FD66c7",
    "usdc": "0xB97EF9Ef8734C71904D8002F8b6Bc66Dd9c48a6E",
    "eurc": "0xc891eb4cbdeff6e073e859e987815ed1505c2acd",
    "rpc": "https://43114.rpc.vialabs.io/",
    "explorer": "https://snowtrace.io/",
    "circleTokenMessenger": "0x6b25532e1060ce10cc3b0a99e5683b91bfde6982",
    "circleMessageTransmitter": "0x8186359af5f57fbb40c6b14a588d2a59c0c29880",
    "circleTokenMinter": "0x420f5035fd5dc62a167e7e7f08b604335ae272b8",
    "isCCTPEnabled": true,
    "featureGateway": "0x869Ad6620D03F7911ff2565C263235Df4D31f0B6",
    "featureCCTP": "0x9fc67B227F4Fae259A64c9C257C39fB2B62867cD",
    "protoCCTPGateway": "0x8888783155201B84613f1F85623eB7625d3B03c9",
    "intentCCTPGateway": "0x0000000000000000000000000000000000000000",
    "cctpLeafChains": [
      {
        "chainId": 13939,
        "rootManager": "0x2a4Ff967F8B18DA6F5264Cc6CE5fAB6BDA47eDe8",
        "leafManager": "0xCc7f1f060990360C91D9887D08aD8497FeDd1605"
      }
    ]
  },
  "48900": {
    "name": "Zircuit Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 48900,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x3b952c8C9C44e8Fe201e2b26F6B2200203214cfF",
    "weth": "0x4200000000000000000000000000000000000006",
    "usdc": "0x3b952c8C9C44e8Fe201e2b26F6B2200203214cfF",
    "protoCCTPGateway": "0xf1A8DEAA78bD956E687df3bfA1115A24ddd3F03d"
  },
  "56026": {
    "name": "Lambda Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 56026,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0x4200000000000000000000000000000000000006",
    "featureGateway": "0x08615FA517d47a9FF8C19F53D5Cbfd2Dd14b554E"
  },
  "59144": {
    "name": "Linea Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 59144,
    "message": "0x6C5AD5e9763Ee118f6c6359e00EeF6ECfA84395b",
    "feeToken": "0x176211869cA2b568f2A7D4EE941E073a821EE1ff",
    "weth": "0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f",
    "rpc": "https://59144.rpc.vialabs.io/",
    "explorer": "https://lineascan.build/"
  },
  "60808": {
    "name": "Bob Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 60808,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xe75d0fb2c24a55ca1e3f96781a2bcc7bdba058f0",
    "weth": "0x4200000000000000000000000000000000000006",
    "featureGateway": "0x08615FA517d47a9FF8C19F53D5Cbfd2Dd14b554E"
  },
  "62050": {
    "name": "Onchain Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 62050,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0x101d0e162933a65cf36ABdD8e29DB93486F35953"
  },
  "70700": {
    "name": "PlayApex Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 70700,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0x77684A04145a5924eFCE0D92A7c4a2A2E8C359de",
    "featureGateway": "0xC705addd35aBa54f17c9E2cf772F829f2f4D9876"
  },
  "111188": {
    "name": "Real Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 111188,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x83feDBc0B85c6e29B589aA6BdefB1Cc581935ECD",
    "weth": "0x90c6E93849E06EC7478ba24522329d14A5954Df4"
  },
  "167000": {
    "name": "Taiko Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 167000,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x07d83526730c7438048D55A4fc0b850e2aaB6f0b",
    "weth": "0xA51894664A773981C6C112C43ce576f315d5b1B6",
    "featureGateway": "0xfDdE8946A0A1755b3a74afE7b3383dEe4fb5712f"
  },
  "200901": {
    "name": "Bitlayer Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 200901,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0xd302be00075a4364df7c53e64638e64fa6a9585c"
  },
  "534352": {
    "name": "Scroll Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 534352,
    "message": "0x6C5AD5e9763Ee118f6c6359e00EeF6ECfA84395b",
    "feeToken": "0x06eFdBFf2a14a7c8E15944D1F4A48F9F95F663A4",
    "weth": "0x5300000000000000000000000000000000000004",
    "rpc": "https://534352.rpc.vialabs.io/",
    "explorer": "https://scrollscan.com/"
  },
  "660279": {
    "name": "Xai Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 660279,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x8712796136Ac8e0EEeC123251ef93702f265aa80",
    "weth": "0x3fB787101DC6Be47cfe18aeEe15404dcC842e6AF",
    "usdc": "0x8712796136Ac8e0EEeC123251ef93702f265aa80",
    "protoCCTPGateway": "0x3012C9175ef181Fb8Da827cc439cd88861cf6aaB"
  },
  "810180": {
    "name": "Zklink Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 810180,
    "message": "0x205B6E9a868F26DC18feE372071C5FA42a6Cea3D",
    "feeToken": "0x1a1A3b2ff016332e866787B311fcB63928464509",
    "weth": "0x8280a4e7D5B3B658ec4580d3Bc30f5e50454F169"
  },
  "984122": {
    "name": "Forma Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 984122,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0xD5eacE1274dBF70960714F513db207433615a263",
    "featureGateway": "0xc6ff38E3dA2B10318E728ec5c097Bed97C1801D8"
  },
  "2702128": {
    "name": "Xterio Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 2702128,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": "0x4200000000000000000000000000000000000006"
  },
  "5151706": {
    "name": "Loot Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 5151706,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "",
    "weth": ""
  },
  "7777777": {
    "name": "Zora Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 7777777,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xCccCCccc7021b32EBb4e8C08314bD62F7c653EC4",
    "weth": "0x4200000000000000000000000000000000000006",
    "featureGateway": "0x48bcCFb7924acf4F944450dCFef2682C72e9758C"
  },
  "10241024": {
    "name": "Alienx Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 10241024,
    "message": "0xf8B8656Ce65Ecf334AcCe299e24E97fB5069c2C8",
    "feeToken": "0x14B90E36Ca85D8B563430C0940E80d3A040285cC",
    "weth": "0xa9b181EB9456170086f9C51AeEcD0511Aa2609e6"
  },
  "20201022": {
    "name": "Pego Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 20201022,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x02F9Bebf5E54968D8Cc2562356C91ECDE135801B",
    "weth": "0x0cF4071940782b640d0b595Cb17bDf3E90869d70"
  },
  "245022934": {
    "name": "Neon Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 245022934,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xEA6B04272f9f62F997F666F07D3a974134f7FFb9",
    "weth": "0x202C35e517Fa803B537565c40F0a6965D7204609",
    "rpc": "https://245022934.rpc.vialabs.io/",
    "explorer": "https://neon.blockscout.com/"
  },
  "666666666": {
    "name": "Degen Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 666666666,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xAa58Cc585b8FC7D6C3c67e9847205AD6B8603929",
    "weth": "0xEb54dACB4C2ccb64F8074eceEa33b5eBb38E5387",
    "featureGateway": "0xC705addd35aBa54f17c9E2cf772F829f2f4D9876"
  },
  "888888888": {
    "name": "Ancient8 Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 888888888,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0x97423A68BAe94b5De52d767a17aBCc54c157c0E5",
    "weth": "0x4200000000000000000000000000000000000006",
    "featureGateway": "0x6EeFCF3fdd03471dbb0dd984C90EA8f678c266Df"
  },
  "1313161554": {
    "name": "Aurora Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1313161554,
    "message": "0xf0861412E46E34DaF7A1fe3fe805e7f713cA3bD9",
    "feeToken": "0xB12BFcA5A55806AaF64E99521918A4bf0fC40802",
    "weth": "0xC9BdeEd33CD01541e1eeD10f90519d2C06Fe3feB",
    "rpc": "https://1313161554.rpc.vialabs.io/",
    "explorer": "https://explorer.aurora.dev/"
  },
  "1380012617": {
    "name": "Rari Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1380012617,
    "message": "0x15AC559DA4951c796DB6620fAb286B96840D039A",
    "feeToken": "0xFbDa5F676cB37624f28265A144A48B0d6e87d3b6",
    "weth": "0xf037540e51D71b2D2B1120e8432bA49F29EDFBD0",
    "featureGateway": "0x75c0223A8f9f8f3baB3FbC6d151381C3C576Ba36"
  },
  "1666600000": {
    "name": "Harmony Mainnet",
    "network": "mainnet",
    "type": "evm",
    "chainId": 1666600000,
    "message": "0x440AA59AC28460a43537a2eA892d96Acd8eaBdb0",
    "feeToken": "0x985458E523dB3d53125813eD68c274899e9DfAb4",
    "weth": "0xcF664087a5bB0237a0BAd6742852ec6c8d69A27a",
    "rpc": "https://1666600000.rpc.vialabs.io/",
    "explorer": "https://explorer.harmony.one/"
  },
  "99999999991": {
    "name": "Solana Mainnet",
    "network": "mainnet",
    "type": "solana",
    "chainId": 99999999991,
    "message": "",
    "feeToken": "",
    "weth": "",
    "usdc": "EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v",
    "eurc": "HzwqbKZw8HxMN6bF2yFZNrht3c2iXXzpKcFu7uBEDKtr",
    "rpc": "",
    "explorer": "https://explorer.solana.com/"
  }
};

export const testnetConfigs: ChainConfigs = {
  "5": {
    "name": "ethereum-goerli",
    "network": "testnet",
    "type": "evm",
    "chainId": 5,
    "message": "0xae65E2211c4119cf92ee85D1a8c4ec20AdaE8aFE",
    "feeToken": "0x2f3bc26eFE51bBe209E0afD2Da29616cF3755E03",
    "weth": "0x95e86440E6B240fc5d476dE44FB56b763B585B2B"
  },
  "31": {
    "name": "rootstock-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 31,
    "message": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f",
    "feeToken": "0x78CC50562941f385c3d0c5204385c3CA686d85b8",
    "weth": "0x8DE416ABd87307f966a5655701F2f78012585225"
  },
  "41": {
    "name": "telos-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 41,
    "message": "0xF8d80d6E52b5B8484a7CD27a5C0F3D35695c57dF",
    "feeToken": "0x2777023E33B88CE622B15eCfA4ADa3cb5a9C545f",
    "weth": "0x02dA7eeEB722A126CCC53085d489ca0e1BAbc55E",
    "usdc": "0x2777023E33B88CE622B15eCfA4ADa3cb5a9C545f",
    "protoCCTPGateway": "0x5aaD587795A268fb0d7e2aBE618704daB13657D4"
  },
  "51": {
    "name": "xdc-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 51,
    "message": "0x00CE686319d401E55c2E7784192EB3A2273Aa9Dc",
    "feeToken": "0x743E00433c33fec5DCF6B9f2cF72d471cd0AB027",
    "weth": "0x49AADBE0f1D2ceb77C7f979b4bF35faEa22976B8"
  },
  "65": {
    "name": "okex-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 65,
    "message": "0x9744D38d26eF45C31c8D20783671506FebeDBAC4",
    "feeToken": "0x2a91b800903BDcC230312C7F96ACF089e6592cBA",
    "weth": "0x967ccdcdf974DCEAb42F20FB61Aafb00D64f0827"
  },
  "71": {
    "name": "conflux-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 71,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "81": {
    "name": "shibuya-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 81,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "83": {
    "name": "meter-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 83,
    "message": "0x01849193F33426C5b9C676757E9324a0f0761E6a",
    "feeToken": "0xDf0F025AA89787A48cfE638CBC053BBF7d93b343",
    "weth": "0xa7719d49246730e2ec6B69aAfC4873E74053b2eB",
    "usdc": "0xDf0F025AA89787A48cfE638CBC053BBF7d93b343",
    "featureGateway": "0x19C7Aa38e6fef4D7Ad92910838A0f9481ba684E9",
    "protoCCTPGateway": "0x1402f37bDe016FcfCFa68c6b2cA222f7C406F1E2"
  },
  "89": {
    "name": "viction-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 89,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "97": {
    "name": "binance-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 97,
    "message": "0x8eF8870CD5583891bDDcf2555e7833bD087392a3",
    "feeToken": "0xD430d3cDFb411810f5E1355C1Be41CF8CCA7e428",
    "weth": "0x768A6c94771EE391C9a996Ff6c30F27190dd0B34",
    "featureGateway": "0xbb3760307d1A5e81dE7639f01F7675EDfc86a773"
  },
  "111": {
    "name": "bob-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 111,
    "message": "0x78CC50562941f385c3d0c5204385c3CA686d85b8",
    "feeToken": "0xAF1f0C79cc043AD4a263dfe7715ca657F9bDeced",
    "weth": "0x17C84459367f9787356b01d7f624Ef0e7E6cA11F"
  },
  "114": {
    "name": "flare-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 114,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "123": {
    "name": "fuse-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 123,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "195": {
    "name": "xlayer-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 195,
    "message": "0x8f554B1b239a57C840d5902D1d901dAFF04F22C2",
    "feeToken": "0x3B142eC3E18890b06bBD893ed4C7dB548E0B6D04",
    "weth": "0xDb52110E8f15a72Ccf3ce205c30C27bdB846057D",
    "usdc": "0x3B142eC3E18890b06bBD893ed4C7dB548E0B6D04",
    "featureGateway": "0xFA954E8B93376Df6aC418510C2f11bCCe8493836",
    "protoCCTPGateway": "0x0F06D07A53C1174a7DC7256Cd6E650abd6aE1e41"
  },
  "282": {
    "name": "cronoszk-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 282,
    "message": "0xB0A4e1Cb4009292Bde2F3282A242FE0eF5C99f03",
    "feeToken": "0x2211725363c2157eF52C2B832A6D0409D3b65975",
    "weth": "0xf6799eCdeCD7F96EA13bc3259852a3a4C47d859E"
  },
  "300": {
    "name": "zksync-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 300,
    "message": "",
    "feeToken": "",
    "weth": ""
  },
  "335": {
    "name": "dfk-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 335,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "338": {
    "name": "cronos-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 338,
    "message": "0x8f92F60ffFB05d8c64E755e54A216090D8D6Eaf9",
    "feeToken": "0xa7fBeDA5DF050bC6c0518A13BE6A2e3A4A2c2175",
    "weth": "0x7EfE948491324f159D9BAfB989Db8ACe8e647685"
  },
  "689": {
    "name": "nero-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 689,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x2777023E33B88CE622B15eCfA4ADa3cb5a9C545f",
    "usdc": "0x2777023E33B88CE622B15eCfA4ADa3cb5a9C545f",
    "weth": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "protoCCTPGateway": "0xC46bc942ca64aed4Eb0B1Af21347944b85EDCb04"
  },
  "701": {
    "name": "fomo-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 701,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x8E3Ea13F2591B5B342FCa9272b59f831476aa74D",
    "weth": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f"
  },
  "842": {
    "name": "taraxa-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 842,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x2777023E33B88CE622B15eCfA4ADa3cb5a9C545f",
    "usdc": "0x2777023E33B88CE622B15eCfA4ADa3cb5a9C545f",
    "weth": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "protoCCTPGateway": "0x4328E74acac8e9BF27BFE53A7cBFDD2217bd0C7b"
  },
  "919": {
    "name": "mode-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 919,
    "message": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f",
    "feeToken": "0x9d75f706b986F0075b3778a12153390273dE95eC",
    "weth": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f"
  },
  "943": {
    "name": "pulse-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 943,
    "message": "0x91e26475016B923527B5Ef15789A9768EBA979e6",
    "feeToken": "0xa0179a4Aa2818ff63Ee6e40b5C27A33BD59e4815",
    "weth": "0xBDc1fCaC4c4615b4FbBE8C401BEd6e29D900fDB0"
  },
  "997": {
    "name": "5ire-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 997,
    "message": "0x8DE416ABd87307f966a5655701F2f78012585225",
    "feeToken": "0x84126A2D8dF65Bb32E6952c351f877E59e02a17f",
    "weth": "0x17C84459367f9787356b01d7f624Ef0e7E6cA11F",
    "usdc": "0x84126A2D8dF65Bb32E6952c351f877E59e02a17f",
    "featureGateway": "0xd72F37d7315Ed8B157628470ae71c6Be89dF9398",
    "protoCCTPGateway": "0x49C34CdC279A486Ea2D2a60cB7F452B888e02ec7"
  },
  "1001": {
    "name": "nexis-zk-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 1001,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "1073": {
    "name": "shimmerevm-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 1073,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "1115": {
    "name": "core-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 1115,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "1287": {
    "name": "moonalpha-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 1287,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "1301": {
    "name": "unichain-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 1301,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x9d75f706b986F0075b3778a12153390273dE95eC",
    "weth": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f"
  },
  "1338": {
    "name": "elysium-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 1338,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0xcfC1d9E275Bf362031E56AA21Eec5E4dD5BD6bb8",
    "usdc": "0xcfC1d9E275Bf362031E56AA21Eec5E4dD5BD6bb8",
    "weth": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "protoCCTPGateway": "0x79D5Ca39d50B9712ac50d0D71a28409Cd8b49471"
  },
  "1401": {
    "name": "dojima-stagenet",
    "network": "stagenet",
    "type": "evm",
    "chainId": 1401,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x9d75f706b986F0075b3778a12153390273dE95eC",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "1452": {
    "name": "gauss-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 1452,
    "message": "0xcbC2d50FA324c187adcf4a186fCb7EcC092E0758",
    "feeToken": "0x5C3293Ff66E77F6FCEefAC24c4766BDcE060B260",
    "weth": "0x0000000000000000000000000000000000000000"
  },
  "1663": {
    "name": "horizen-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 1663,
    "message": "0xA95c0BC77Ab8a8EfA3dF00366FFAe5CB1A2cba15",
    "feeToken": "0x17C84459367f9787356b01d7f624Ef0e7E6cA11F",
    "weth": "0x291BcefF63A9A2726160391928EFCEe34014d58A"
  },
  "1945": {
    "name": "onus-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 1945,
    "message": "0xF8d80d6E52b5B8484a7CD27a5C0F3D35695c57dF",
    "feeToken": "0x5f4d59B985959b1dd5da23C574228e9161D34845",
    "weth": "0x02dA7eeEB722A126CCC53085d489ca0e1BAbc55E"
  },
  "1946": {
    "name": "soneium-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 1946,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x9d75f706b986F0075b3778a12153390273dE95eC",
    "weth": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f"
  },
  "1992": {
    "name": "sanko-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 1992,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "1998": {
    "name": "kyoto-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 1998,
    "message": "0xdCa897f920Df8015169838c428479D5e3d5Bf526",
    "feeToken": "0x3a60C03a86eEAe30501ce1af04a6C04Cf0188700",
    "weth": "0x73Db0a9a3aB2098B57E6A0f1F8Ff497e4adaa412"
  },
  "2021": {
    "name": "saigon-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 2021,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "2047": {
    "name": "stratos-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 2047,
    "message": "0x4a7B33299a21c518d77eb3fF00fd1DC39C452Cba",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x6134465958b5FBF8812fE743534C66D833Fa9046"
  },
  "2221": {
    "name": "kava-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 2221,
    "message": "0xd577fcBee5734c2da5e0063fF1df38845DaA7117",
    "feeToken": "0x1Ec7Dfbc9e310768A17145f03f3451f562cEc773",
    "weth": "0xc5f8fbE61aF2b5f0DB8A51e10133062e54F77947"
  },
  "2371": {
    "name": "nexis-devnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 2371,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "2442": {
    "name": "polygonzk-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 2442,
    "message": "",
    "feeToken": "",
    "weth": ""
  },
  "2522": {
    "name": "fraxtal-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 2522,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f",
    "weth": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f"
  },
  "2888": {
    "name": "boba-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 2888,
    "message": "0x1Ec7Dfbc9e310768A17145f03f3451f562cEc773",
    "feeToken": "0x8cd2556e02f5A136167aE123e313a17481f51904",
    "weth": "0x02dA7eeEB722A126CCC53085d489ca0e1BAbc55E"
  },
  "3939": {
    "name": "dos-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 3939,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "4002": {
    "name": "fantom-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 4002,
    "message": "0x48964a49B5826DB6Cb8d8ed0dAf93eEeD734b923",
    "feeToken": "0xAC78952AD62A468ac21f43DfA8B14f2c8Be87582",
    "weth": "0x9364b2FC6f07c123dB5a398aE655DFCf7F445d54"
  },
  "4460": {
    "name": "orderly-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 4460,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "4690": {
    "name": "iotex-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 4690,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "5003": {
    "name": "mantle-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 5003,
    "message": "0x9d75f706b986F0075b3778a12153390273dE95eC",
    "feeToken": "0xc2e7eA22A69Fff2ac96EB9b306A562C9fb299b16",
    "weth": "0x78CC50562941f385c3d0c5204385c3CA686d85b8"
  },
  "5115": {
    "name": "citrea-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 5115,
    "message": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "feeToken": "0x28d03e97e68Fb138a2318A2627806dfAF9972755",
    "usdc": "0x28d03e97e68Fb138a2318A2627806dfAF9972755",
    "weth": "0xc2e7eA22A69Fff2ac96EB9b306A562C9fb299b16",
    "protoCCTPGateway": "0x7bB78097d7e672D65cD6596Ee9b4097CE16AC391"
  },
  "5611": {
    "name": "opbnb-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 5611,
    "message": "0x7bB78097d7e672D65cD6596Ee9b4097CE16AC391",
    "feeToken": "0x17C84459367f9787356b01d7f624Ef0e7E6cA11F",
    "weth": "0x6134465958b5FBF8812fE743534C66D833Fa9046",
    "circleTokenMessenger": "0x9f3B8679c73C2Fef8b59B4f3444d4e156fb70AA5",
    "circleMessageTransmitter": "0x7865fAfC2db2093669d92c0F33AeEF291086BEFD"
  },
  "6911": {
    "name": "nibiru-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 6911,
    "message": "0x0eefCF172F7e5C04A8d565f4e955968221fDb18f",
    "usdc": "0x500161413B77a8a350755A9d774D3b23B89a4b8B",
    "feeToken": "0x500161413B77a8a350755A9d774D3b23B89a4b8B",
    "weth": "0xF8d80d6E52b5B8484a7CD27a5C0F3D35695c57dF",
    "protoCCTPGateway": "0x5D1FB3688888E6F00f398490BAc4B45f05c3F425"
  },
  "7001": {
    "name": "zetachain-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 7001,
    "message": "0x714c9202B3B5AF0C0Ad844c2a71803cebBFD3AF5",
    "feeToken": "0x8cd2556e02f5A136167aE123e313a17481f51904",
    "weth": "0xbd9F6E783a2872f703953F0db1d6D60912F86af4"
  },
  "7210": {
    "name": "nibiru-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 7210,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0xdBe060aAF22aE343C19A3Be553ddE0623125e552",
    "usdc": "0xdBe060aAF22aE343C19A3Be553ddE0623125e552",
    "weth": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "protoCCTPGateway": "0x135b709b4C646C203bcB669578C5F9345b9882fE"
  },
  "7222": {
    "name": "nibiru-devnet3",
    "network": "testnet",
    "type": "evm",
    "chainId": 7222,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x3C615A128acb9E3247d4CF24CaA0a987d3816754",
    "weth": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f"
  },
  "7701": {
    "name": "canto-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 7701,
    "message": "0x09FC1B8e1651A0D35258Ab919035d3087245F8f3",
    "feeToken": "0xeDCfb3038d4a262Aef3735CFa64146C35196CC51",
    "weth": "0x95e86440E6B240fc5d476dE44FB56b763B585B2B"
  },
  "7771": {
    "name": "bitrock-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 7771,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "9372": {
    "name": "oasys-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 9372,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "9768": {
    "name": "mainnetz-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 9768,
    "message": "0x714c9202B3B5AF0C0Ad844c2a71803cebBFD3AF5",
    "feeToken": "0x714853D6197e560013ee161fC259b87E8B3cA7E9",
    "weth": "0xCdc7D21D3F75809ec8CC3030AB8018A98AD0b296"
  },
  "10081": {
    "name": "japan-open-chain-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 10081,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "10200": {
    "name": "gnosis-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 10200,
    "message": "0x88776c0FbaCA594938C6B87a42a69D530A8CCDF3",
    "feeToken": "0x3ecDbd62D72917b4D22b638E0a550cA516cfe139",
    "weth": "0x88d4E6cb8955e5b406dbac9e4BCd101082947433"
  },
  "13473": {
    "name": "immutable-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 13473,
    "message": "0xC7E87B6614DAb7a4B3Feaa9e56a2cA29A84AD0a8",
    "feeToken": "0xDb126ef7666c97db7D9F70FFbCfFdC567b3B25A5",
    "weth": "0xfA8d81973f29b2555933027Ac9751740B9fFa047"
  },
  "13939": {
    "name": "reef-testnet",
    "network": "testnet",
    "type": "reef",
    "chainId": 13939,
    "message": "0xf7af4C3ecA8Dd2E2a59eD44060537a09f701C6ed",
    "messageSubstrate": "5EMjsd1BodbqwQ6S71CigZuz2oprqP9GQmPJDVKZCCF1cKc6",
    "feeToken": "0x72761eBBA6395D6deE4A8e273c1E487a72693c46",
    "weth": "0x0000000000000000000000000000000001000000"
  },
  "14333": {
    "name": "vitruveo-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 14333,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "17000": {
    "name": "ethereum-holesky",
    "network": "testnet",
    "type": "evm",
    "chainId": 17000,
    "message": "0x668de98389d5d6C9064E40Cfda2FC6471EDDE7ff",
    "feeToken": "0x6900384BA33f8C635DeE2C3BD7d46A0626FfB096",
    "weth": "0xB09780607F0cbCC3c7B95f17d42417269ED53a1D",
    "usdc": "",
    "circleTokenMessenger": ""
  },
  "18233": {
    "name": "real-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 18233,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "23295": {
    "name": "oasis-sapphire-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 23295,
    "message": "0x9c90eC23162C818A79B46C79Bb6EBC07C6733919",
    "feeToken": "0x4B5b1163525A6Cebd9a06F4C386976F2B41A4Bdf",
    "weth": "0xc9cd1956655B8085b9e53dA7F5813B75B95F69d3"
  },
  "42069": {
    "name": "gelato-op-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 42069,
    "message": "0xAede7a77D49Eb88Cb129896d69f0E66ee51D44AC",
    "feeToken": "0xc45c3cEc0C800fCB95636D1FB6ec0AE3d42b58e2",
    "weth": "0x765d2951bF4d512a5FDbD6384C4b6aD5ab1b1873"
  },
  "42261": {
    "name": "oasis-emerald-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 42261,
    "message": "0x9Ca377D441B01A44fEab8D75B992ab2e4f710BA9",
    "feeToken": "0x73Db0a9a3aB2098B57E6A0f1F8Ff497e4adaa412",
    "weth": "0x0000000000000000000000000000000000000000"
  },
  "43113": {
    "name": "avalanche-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 43113,
    "message": "0x8f92F60ffFB05d8c64E755e54A216090D8D6Eaf9",
    "feeToken": "0x5425890298aed601595a70ab815c96711a31bc65",
    "weth": "0xD59A1806BAa7f46d1e07A07649784fA682708794",
    "usdc": "0x5425890298aed601595a70ab815c96711a31bc65",
    "rpc": "https://43113.rpc.vialabs.io/",
    "explorer": "https://testnet.snowtrace.io/",
    "featureGateway": "0x70B806CBE98953befd43c558E70AeD133f98A44B",
    "featureCCTP": "0x5EdA91d22B31A0eD5717295fCBA0C5E990Fdf1B9",
    "circleTokenMessenger": "0xeb08f243e5d3fcff26a9e38ae5520a669f4019d0",
    "circleMessageTransmitter": "0xa9fb1b3009dcb79e2fe346c16a604b8fa8ae0a79",
    "circleTokenMinter": "0x4ed8867f9947a5fe140c9dc1c6f207f3489f501e",
    "protoCCTPGateway": "0x02986E15f847F4dc509F01B781E20F95da851b44",
    "isCCTPEnabled": true,
    "cctpLeafChains": [
      {
        "chainId": 64165,
        "rootManager": "0xD882a92421490aD0e500043B7E60506F45bE6716",
        "leafManager": "0x3EdCD98439d8fc68Fc46825C97880Af9eF4b0272"
      },
      {
        "chainId": 195,
        "rootManager": "0x60fB013997CFa53729A9c8Dfbacecc65c8Dde897",
        "leafManager": "0xa06f5842bB1bF8ea42397fb116BFA17c3F8c8c5b"
      },
      {
        "chainId": 779672,
        "rootManager": "0xA9256742d8240E986AaBE6d9A594da914E425785",
        "leafManager": "0x1db51fEfc5c67E2C6D5A5BA18507F762b9715C43"
      },
      {
        "chainId": 173750,
        "rootManager": "0xad1A3c80492e6E1e1ad115020B0270EEc4C52457",
        "leafManager": "0x28d03e97e68Fb138a2318A2627806dfAF9972755"
      }
    ]
  },
  "43851": {
    "name": "zkfair-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 43851,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "44787": {
    "name": "celo-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 44787,
    "message": "0xc959284fae7Cc3F41367dA2Df595b7267597094C",
    "feeToken": "0x2A07454Bc8177b944fBdB4575a4b0bd040a70e95",
    "weth": "0x3BC1695644Ad2c60f1500cea16A9536AF5DDd38e"
  },
  "48899": {
    "name": "zircuit-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 48899,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0xA1AeC40c1B58226407B8b0dF050c8D9ab0df7a7D",
    "weth": "0x9d75f706b986F0075b3778a12153390273dE95eC",
    "usdc": "0xA1AeC40c1B58226407B8b0dF050c8D9ab0df7a7D",
    "featureGateway": "0x1D006355e6778d06a40296eE510bbc09f60c9e1c",
    "protoCCTPGateway": "0xefbdB4f11b61AEb417AAC34504E9535f5218A8f2"
  },
  "57000": {
    "name": "rollux-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 57000,
    "message": "0x7bB78097d7e672D65cD6596Ee9b4097CE16AC391",
    "feeToken": "0x17C84459367f9787356b01d7f624Ef0e7E6cA11F",
    "weth": "0x6134465958b5FBF8812fE743534C66D833Fa9046"
  },
  "59140": {
    "name": "linea-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 59140,
    "message": "0x0eefCF172F7e5C04A8d565f4e955968221fDb18f",
    "feeToken": "0xF8d80d6E52b5B8484a7CD27a5C0F3D35695c57dF",
    "weth": "0x167Be5395286327da2E376f3DF2bF46ED42209B0"
  },
  "59902": {
    "name": "metis-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 59902,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "64165": {
    "name": "sonic-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 64165,
    "message": "0xeFaDc14c2DD95D0E6969d0B25EA6e4F830150493",
    "feeToken": "0x67C375093cc98D6E56f2fAF0C3D3f75d3a088499",
    "weth": "0xdCa897f920Df8015169838c428479D5e3d5Bf526",
    "usdc": "0x67C375093cc98D6E56f2fAF0C3D3f75d3a088499",
    "featureGateway": "0x64555eE18ea829d6d0f08527EF86eFD8d9BA8A73",
    "protoCCTPGateway": "0x9DA7e3112ccE2Da5C5ea31ED6E1BbdC328d7Da23"
  },
  "80002": {
    "name": "polygon-amoy",
    "network": "testnet",
    "type": "evm",
    "chainId": 80002,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x41e94eb019c0762f9bfcf9fb1e58725bfb0e7582",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f",
    "usdc": "0x41e94eb019c0762f9bfcf9fb1e58725bfb0e7582",
    "rpc": "https://80002.rpc.vialabs.io/",
    "explorer": "https://mumbai.polygonscan.com/",
    "featureGateway": "0x9Fe535E31A20A9dFe5D66CE47dbb2fB69ea63Ab9",
    "featureCCTP": "0x2661237BFEc9d09b7cbca3AA01361f1C2650705d",
    "circleTokenMessenger": "0x9f3B8679c73C2Fef8b59B4f3444d4e156fb70AA5",
    "circleMessageTransmitter": "0x7865fAfC2db2093669d92c0F33AeEF291086BEFD",
    "circleTokenMinter": "0xE997d7d2F6E065a9A93Fa2175E878Fb9081F1f0A",
    "protoCCTPGateway": "0x9803cdfd229ac9c33839F4aAF4a13A276a789c24",
    "isCCTPEnabled": true,
    "cctpLeafChains": [
      {
        "chainId": 65100004,
        "rootManager": "0x1EdAbc528261A942e58eD1e02E4c5C849a9F7cdD",
        "leafManager": "0x9d1421d5813f6ba074880472D5775e820e18Fd7A"
      }
    ]
  },
  "84532": {
    "name": "base-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 84532,
    "message": "0xE700Ee5d8B7dEc62987849356821731591c048cF",
    "feeToken": "0x036CbD53842c5426634e7929541eC2318f3dCF7e",
    "weth": "0x32D9c1DA01F221aa0eab4A0771Aaa8E2344ECd35",
    "usdc": "0x036CbD53842c5426634e7929541eC2318f3dCF7e",
    "rpc": "https://84532.rpc.vialabs.io/",
    "explorer": "https://sepolia.basescan.org/",
    "featureGateway": "0x9Fc203b5dc5c29912D5c2BEF6ADBC6C8d783ea9B",
    "featureCCTP": "0x2253C9B38aCE84c189FbbCc25535dbA5A0015Ef0",
    "circleTokenMessenger": "0x9f3B8679c73C2Fef8b59B4f3444d4e156fb70AA5",
    "circleMessageTransmitter": "0x7865fAfC2db2093669d92c0F33AeEF291086BEFD",
    "circleTokenMinter": "0xE997d7d2F6E065a9A93Fa2175E878Fb9081F1f0A",
    "protoCCTPGateway": "0xdcc8769Be2F2E938F02f66e9F8Bb224a81da5Bc9",
    "isCCTPEnabled": true,
    "cctpLeafChains": [
      {
        "chainId": 83,
        "rootManager": "0x0553dDfc9A01D200B8608158295157fcBC63479d",
        "leafManager": "0xF58725d7a4c0fBf7978A93772F284e8019Ee8E53"
      },
      {
        "chainId": 325000,
        "rootManager": "0xdBe060aAF22aE343C19A3Be553ddE0623125e552",
        "leafManager": "0x9d1421d5813f6ba074880472D5775e820e18Fd7A"
      },
      {
        "chainId": 41,
        "rootManager": "0xF3Db272879071B64C5a81499b07d2617974dE6B7",
        "leafManager": "0x9d1421d5813f6ba074880472D5775e820e18Fd7A"
      },
      {
        "chainId": 1338,
        "rootManager": "0x500161413B77a8a350755A9d774D3b23B89a4b8B",
        "leafManager": "0xa1cbbf90A38B7C279a1Ef17E29cC61968AaA6945"
      },
      {
        "chainId": 842,
        "rootManager": "0x92D758eD7f324f1821AD01baD68679aE32376110",
        "leafManager": "0x9d1421d5813f6ba074880472D5775e820e18Fd7A"
      }
    ]
  },
  "128123": {
    "name": "etherlink-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 128123,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f",
    "usdc": "",
    "circleTokenMessenger": "",
    "circleMessageTransmitter": ""
  },
  "167008": {
    "name": "katla-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 167008,
    "message": "0xA95c0BC77Ab8a8EfA3dF00366FFAe5CB1A2cba15",
    "feeToken": "0x17C84459367f9787356b01d7f624Ef0e7E6cA11F",
    "weth": "0x291BcefF63A9A2726160391928EFCEe34014d58A"
  },
  "167009": {
    "name": "taiko-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 167009,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "173750": {
    "name": "avalanche-echo-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 173750,
    "message": "0x0000000000000000000000000000000000000000",
    "feeToken": "0x2777023E33B88CE622B15eCfA4ADa3cb5a9C545f",
    "usdc": "0x2777023E33B88CE622B15eCfA4ADa3cb5a9C545f",
    "weth": "0x8228407F8bc38790997616B18AF8773016270558",
    "protoCCTPGateway": "0x3C615A128acb9E3247d4CF24CaA0a987d3816754"
  },
  "325000": {
    "name": "camp-network-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 325000,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x2777023E33B88CE622B15eCfA4ADa3cb5a9C545f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f",
    "usdc": "0x2777023E33B88CE622B15eCfA4ADa3cb5a9C545f",
    "protoCCTPGateway": "0x059D91b3C7111e82CEf4326Fa78b365045c42b79"
  },
  "421614": {
    "name": "arbitrum-sepolia",
    "network": "testnet",
    "type": "evm",
    "chainId": 421614,
    "message": "0x0D7e59B0390e47E6C3a29cCdF68e43f3e50e2119",
    "feeToken": "0x75faf114eafb1BDbe2F0316DF893fd58CE46AA4d",
    "weth": "0x47963cB18B1aef899efcdC5EF7341B5167e5d4E0",
    "usdc": "0x75faf114eafb1BDbe2F0316DF893fd58CE46AA4d",
    "rpc": "https://421614.rpc.vialabs.io/",
    "explorer": "https://sepolia.arbiscan.io/",
    "featureGateway": "0x176c9Eb7aabC9A9857E6a08980b463b6C6FE0486",
    "featureCCTP": "0x761fC9166a95f686e55B66C397d04C0889c5c92F",
    "circleTokenMessenger": "0x9f3B8679c73C2Fef8b59B4f3444d4e156fb70AA5",
    "circleMessageTransmitter": "0xaCF1ceeF35caAc005e15888dDb8A3515C41B4872",
    "circleTokenMinter": "0xE997d7d2F6E065a9A93Fa2175E878Fb9081F1f0A",
    "protoCCTPGateway": "0x040F70B724F1E7f8509848e750bbF10e20b73f60",
    "isCCTPEnabled": true,
    "cctpLeafChains": [
      {
        "chainId": 997,
        "rootManager": "0x448811d70486296d4e2B60E4258328957C94EaA7",
        "leafManager": "0x28d03e97e68Fb138a2318A2627806dfAF9972755"
      },
      {
        "chainId": 37714555429,
        "rootManager": "0x46bFafc74f8dD1FC4c8Ea963ac881504Ba75D5fD",
        "leafManager": "0x9d1421d5813f6ba074880472D5775e820e18Fd7A"
      },
      {
        "chainId": 5115,
        "rootManager": "0x610F32033567097454cE0DF669Abc949Fb2f256B",
        "leafManager": "0x2660b88B931aE3b8dBcb796F3038305C46eb7F88"
      },
      {
        "chainId": 7210,
        "rootManager": "0xdc61e66FaFc8D7F1293D8d31C2068D2E4692428D",
        "leafManager": "0xAD1837d07c5e4396cD57491Fd3B4eCfa3aD7f5E5"
      },
      {
        "chainId": 689,
        "rootManager": "0xad1A3c80492e6E1e1ad115020B0270EEc4C52457",
        "leafManager": "0x9d1421d5813f6ba074880472D5775e820e18Fd7A"
      },
      {
        "chainId": 6911,
        "rootManager": "0xf17a03bD8d9F641Ff2344B9FB5d786DDb62D09B8",
        "leafManager": "0xFb78F049683D3EdC0808a03297df9da8f2633315"
      }
    ]
  },
  "534351": {
    "name": "scroll-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 534351,
    "message": "0x8Dcb34c02365116565A3d68b97e4ae98F983B9D0",
    "feeToken": "0x08f0A528dC83f4Db2C7B05478ED3B098eCE94fCB",
    "weth": "0xD27f1d1093162560A7e17b7fd3A7711Caf6cE666"
  },
  "686868": {
    "name": "merlin-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 686868,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "713715": {
    "name": "sei-devnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 713715,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "779672": {
    "name": "avalanche-dispatch-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 779672,
    "message": "0x0000000000000000000000000000000000000000",
    "feeToken": "0x0e80E22b69C48CcDC54B4D0A24D497d8203B7b1E",
    "usdc": "0x0e80E22b69C48CcDC54B4D0A24D497d8203B7b1E",
    "weth": "0xA68E53100d11611F11A8956818F9Bd3DD3A2eE8d",
    "protoCCTPGateway": "0x144c21008a6e31291aAADec4CAc09c73bc640Fb4"
  },
  "1261120": {
    "name": "gelato-zkatana-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 1261120,
    "message": "0xAede7a77D49Eb88Cb129896d69f0E66ee51D44AC",
    "feeToken": "0xc45c3cEc0C800fCB95636D1FB6ec0AE3d42b58e2",
    "weth": "0x765d2951bF4d512a5FDbD6384C4b6aD5ab1b1873"
  },
  "3441006": {
    "name": "manta-pacific-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 3441006,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "11155111": {
    "name": "sepolia-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 11155111,
    "message": "0xF2AA17F92d4D9Be1c0b0D2A3235A06143A2D729f",
    "feeToken": "0x1c7D4B196Cb0C7B01d743Fbc6116a902379C7238",
    "weth": "0x6c884e8b8139a87A68b7c4350a9a25305f6de0b6",
    "usdc": "0x1c7D4B196Cb0C7B01d743Fbc6116a902379C7238",
    "rpc": "https://11155111.rpc.vialabs.io/",
    "explorer": "https://sepolia.etherscan.io/",
    "featureGateway": "0xbf5288c8e8d5A1b89EFC2CefF1836f4c1392507f",
    "featureCCTP": "0x987177924594a83b93aD04C6A57BFB59a78f632E",
    "circleTokenMessenger": "0x9f3B8679c73C2Fef8b59B4f3444d4e156fb70AA5",
    "circleMessageTransmitter": "0x7865fAfC2db2093669d92c0F33AeEF291086BEFD",
    "circleTokenMinter": "0xE997d7d2F6E065a9A93Fa2175E878Fb9081F1f0A",
    "protoCCTPGateway": "0xC46bc942ca64aed4Eb0B1Af21347944b85EDCb04",
    "isCCTPEnabled": true,
    "cctpLeafChains": [
      {
        "chainId": 48899,
        "rootManager": "0xC7c54ca043dC8e5dd1dF6E5a4B4A2Df42A1ba59d",
        "leafManager": "0x31c3E0eeF8412aC4dF12b5C5B0FF25cff2aBA65A"
      }
    ]
  },
  "11155420": {
    "name": "optimism-sepolia",
    "network": "testnet",
    "type": "evm",
    "chainId": 11155420,
    "message": "0xe511183765E1F325702EF8F3d92046e9d6DF6742",
    "feeToken": "0x5fd84259d66Cd46123540766Be93DFE6D43130D7",
    "weth": "0x3574311D950E04A48289DA64759016c26725B180",
    "usdc": "0x5fd84259d66Cd46123540766Be93DFE6D43130D7",
    "rpc": "https://11155420.rpc.vialabs.io/",
    "explorer": "https://sepolia-optimism.etherscan.io/",
    "featureGateway": "0x5f44Da08068cb232F8Fa7F730017079A1d06B800",
    "featureCCTP": "0xc1c93f3A19C70780308529d39BE1f8786ECd8CC9",
    "circleTokenMessenger": "0x9f3B8679c73C2Fef8b59B4f3444d4e156fb70AA5",
    "circleMessageTransmitter": "0x7865fAfC2db2093669d92c0F33AeEF291086BEFD",
    "circleTokenMinter": "0xE997d7d2F6E065a9A93Fa2175E878Fb9081F1f0A",
    "protoCCTPGateway": "0x0bD2dDddb088703F139a6d5a1dF0A25120607907",
    "isCCTPEnabled": true
  },
  "12227332": {
    "name": "neox-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 12227332,
    "message": "",
    "feeToken": "",
    "weth": ""
  },
  "65100003": {
    "name": "autonity-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 65100003,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x3a60C03a86eEAe30501ce1af04a6C04Cf0188700",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "65100004": {
    "name": "autonity-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 65100004,
    "message": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f",
    "feeToken": "0x2777023E33B88CE622B15eCfA4ADa3cb5a9C545f",
    "usdc": "0x2777023E33B88CE622B15eCfA4ADa3cb5a9C545f",
    "weth": "0x78CC50562941f385c3d0c5204385c3CA686d85b8",
    "protoCCTPGateway": "0xAD1837d07c5e4396cD57491Fd3B4eCfa3aD7f5E5"
  },
  "65110000": {
    "name": "autonity-devnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 65110000,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0xf92De0c98007EBaDa22E138A881710B82C513Da0",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "68840142": {
    "name": "frame-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 68840142,
    "message": "0xCb69924aDf996315aDcd9051ccE2B572dD9450a9",
    "feeToken": "0x71311899492757cD512a11855C150EA219bB3A42",
    "weth": "0xf25D6E21554a5503DEA695acdad7586778a89C39"
  },
  "111557560": {
    "name": "cyber-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 111557560,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "123420111": {
    "name": "gelato-op-celestia-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 123420111,
    "message": "0xAede7a77D49Eb88Cb129896d69f0E66ee51D44AC",
    "feeToken": "0xc45c3cEc0C800fCB95636D1FB6ec0AE3d42b58e2",
    "weth": "0x765d2951bF4d512a5FDbD6384C4b6aD5ab1b1873"
  },
  "168587773": {
    "name": "blast-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 168587773,
    "message": "0xA95c0BC77Ab8a8EfA3dF00366FFAe5CB1A2cba15",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x765d2951bF4d512a5FDbD6384C4b6aD5ab1b1873"
  },
  "245022926": {
    "name": "neon-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 245022926,
    "message": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "feeToken": "0x512E48836Cd42F3eB6f50CEd9ffD81E0a7F15103",
    "weth": "0x11adC2d986E334137b9ad0a0F290771F31e9517F"
  },
  "999999999": {
    "name": "zora-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 999999999,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "1313161555": {
    "name": "aurora-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 1313161555,
    "message": "0x52e1CFE18BD55bb8d885d463DC26D9C365cd316B",
    "feeToken": "0x1D006355e6778d06a40296eE510bbc09f60c9e1c",
    "weth": "0xA648df7adFa847Cf0149083857ddA1680926899F"
  },
  "1530162493": {
    "name": "via-kaleido-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 1530162493,
    "message": "",
    "feeToken": "",
    "weth": ""
  },
  "1666700000": {
    "name": "harmony-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 1666700000,
    "message": "0x9cAa65b69Ad8118C3d1454393F5b96292FE3C0aB",
    "feeToken": "0x3574311D950E04A48289DA64759016c26725B180",
    "weth": "0xbF1Bb50a9607d2191aB4e5925B48f44306000A05"
  },
  "1918988905": {
    "name": "rari-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 1918988905,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x566B40Dd59A868c244E1353368e08ddaD1C1d74f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f"
  },
  "37714555429": {
    "name": "xai-testnet",
    "network": "testnet",
    "type": "evm",
    "chainId": 37714555429,
    "message": "0x3B5b764229b2EdE0162220aF51ab6bf7f8527a4F",
    "feeToken": "0x2777023E33B88CE622B15eCfA4ADa3cb5a9C545f",
    "weth": "0x0EFafca24E5BbC1C01587B659226B9d600fd671f",
    "usdc": "0x2777023E33B88CE622B15eCfA4ADa3cb5a9C545f",
    "featureGateway": "0x06EF0100f5CA88E6F1F5742BD3a913D29Db39505",
    "protoCCTPGateway": "0x84b8Ffb85f89665DA3c349A0D707396EE492A20e"
  }
};

export const chains: {
    mainnet: ChainConfigs;
    testnet: ChainConfigs;
} = {
    mainnet: mainnetConfigs,
    testnet: testnetConfigs
};

export function getChainConfig(chainId: number): ChainConfig | undefined {
    return mainnetConfigs[chainId] || testnetConfigs[chainId];
}

export function isTestnet(chainId: number): boolean {
    return !!testnetConfigs[chainId];
}

export function getProtoCCTPGateway(chainId: number): string | undefined {
    const config = getChainConfig(chainId);
    return config?.protoCCTPGateway;
}

export function getIntentCCTPGateway(chainId: number): string | undefined {
    const config = getChainConfig(chainId);
    return config?.intentCCTPGateway;
}

export function getCCTPLeafChains(chainId: number): ChainConfig['cctpLeafChains'] {
    const config = getChainConfig(chainId);
    return config?.cctpLeafChains;
}
